exonum-java-binding/core/src/main/java/com/exonum/binding/core/storage/indices/ProofMapIndexProxy.java

```diff
@@ -85,7 +85,8 @@
  * @param <V> the type of values in this map
  * @see View
  */
-public final class ProofMapIndexProxy<K, V> extends AbstractIndexProxy implements MapIndex<K, V> {
+public final class ProofMapIndexProxy<K, V> extends AbstractIndexProxy implements MapIndex<K, V>,
```

Comment:
When we make the index factories and add the interfaces for each index type, those interfaces could extend HashableIndex, instead of implementation classes.